src/lib/minions/functions/removeFoodFromUser.ts

```diff
@@ -0,0 +1,56 @@
+import { KlasaClient, KlasaUser } from 'klasa';
+import { addBanks, removeBankFromBank } from 'oldschooljs/dist/util';
+import { O } from 'ts-toolbelt';
+
+import { Eatables } from '../../eatables';
+import { ClientSettings } from '../../settings/types/ClientSettings';
+import { UserSettings } from '../../settings/types/UserSettings';
+import { Bank, ItemBank } from '../../types';
+
+export function getUserFoodFromBank(
```

Comment:
youre exporting 2 functions from this file, when the file is named after only 1, just put the other in a new file